src/Tasks/Common/Resources/xlf/Strings.pt-BR.xlf

```diff
@@ -357,6 +357,11 @@
         <target state="new">You are working with a preview version of the .NET Core SDK. You can define the SDK version via a global.json file in the current project. More at https://docs.microsoft.com/en-us/dotnet/core/tools/global-json</target>
         <note />
       </trans-unit>
+      <trans-unit id="AssetsFileNotSet">
+        <source>ProjectAssetsFile was not set. Run a NuGet package restore to generate this file.</source>
+        <target state="new">ProjectAssetsFile was not set. Run a NuGet package restore to generate this file.</target>
+        <note>"ProjectAssetsFile" is the name of a property and parameter to the task, so it should not be translated</note>
```

Comment:
This is a common enough case, right? e.g. You'd get it using `msbuild`  without /restore. I don't think we should be mentioning ProjectAssetsFile the variable, but rather the fact that we can't locate the project assets file. This is getting into the implementation details.